source/Applications/Basic/Visualization/CaptureWritePCLVis3D/CaptureWritePCLVis3D.cpp

```diff
@@ -11,8 +11,126 @@ save it to PCD file format, and visualize it.
 #include <pcl/visualization/cloud_viewer.h>
 
 #include <iostream>
+#include <thread>
 
-int main()
+namespace
+{
+    void visualizePointCloud(const Zivid::PointCloud &pointCloud)
+    {
+        std::cout << "Setting up visualization" << std::endl;
+        Zivid::Visualization::Visualizer visualizer;
+
+        std::cout << "Visualizing point cloud" << std::endl;
+        visualizer.showMaximized();
+        visualizer.show(pointCloud);
+        visualizer.resetToFit();
+
+        std::cout << "Running visualizer. Blocking until window closes" << std::endl;
+        visualizer.run();
+    }
```

Comment:
I think remove the Zivid visualization here, since the sample https://github.com/zivid/zivid-cpp-samples/blob/master/source/Applications/Basic/Visualization/CaptureVis3D/CaptureVis3D.cpp is for that

Reply:
And remove corresponding header and linking to Zivid visualizer

Reply:
done